boards/arm/disco_l475_iot1/pinmux.c

```diff
@@ -43,7 +43,7 @@ static const struct pin_config pinconf[] = {
 #if DT_NODE_HAS_STATUS(DT_NODELABEL(spi3), okay) && CONFIG_SPI
 	/* SPI3 is used for BT/WIFI, Sub GHZ communication */
 	{STM32_PIN_PC10, STM32L4X_PINMUX_FUNC_PC10_SPI3_SCK},
-	{STM32_PIN_PC11, STM32L4X_PINMUX_FUNC_PC11_SPI3_MISO | \
+	{STM32_PIN_PC11, STM32L4X_PINMUX_FUNC_PC11_SPI3_MISO |
 		STM32_OSPEEDR_VERY_HIGH_SPEED},
```

Comment:
Do we need this change ?

Reply:
yes,'checkpatch' asks for

Reply:
removed : no change on boards/arm/disco_l475_iot1/pinmux.c